Documentation/compilation.rst

```diff
@@ -87,9 +87,25 @@ Installing Libxsmm
    cp bin/libxsmm_gemm_generator $HOME/bin
    cd ..
 
+
+Installing ImpalaJIT (relevant only for CMake)
+----------------------------------------------
+
+.. code-block:: bash
+ 
+   git clone https://github.com/uphoffc/ImpalaJIT.git
+   cd ImpalaJIT
+   mkdir build
+   cd build 
+   cmake .. -DCMAKE_INSTALL_PREFIX=$HOME
+   make
+   make install
+
+
```

Comment:
We should move the cmake config to a separate file. Could you please start this (with a huge warning DO NOT USE)? I would then write the rest of the documentation :)